Geometry/CMSCommonData/python/cmsExtendedGeometryDetailedCavern2018XML_cfi.py

```diff
@@ -239,10 +239,10 @@
         'Geometry/MuonCommonData/data/mfshield/2017/v1/mfshield.xml',
         'Geometry/MuonCommonData/data/gemf/TDR_BaseLine/gemf.xml',
         'Geometry/MuonCommonData/data/gem11/2017/v2/gem11.xml',
+        'Geometry/ForwardCommonData/data/forward/2018/v1/forward.xml',
```

Comment:
@ianna @civanch the addition of CASTOR should be relevant only for HI data, right?

Reply:
Yes. Unfortunately, there are some modules which ask for Castor hits in any case. Given that the production stops when they are not available, the easiest way was to keep it in the geometry description. The alternative would be to customize the producers, but it's not in my domain.